.gitignore

```diff
@@ -23,3 +23,6 @@ user.props
 
 # MacOS desktop metadata files
 .DS_Store
+
+# If someone downloads uncrustify to be installed via homebrew.
+uncrustify.rb
```

Comment:
is this needed?  (the command below does a `rm uncrustify.rb`)

Reply:
not strictly needed; just figured no harm including it here (unless you want to keep `.gitignore` as barebones as possible). A safety-conscientious individual may want to inspect `uncrustify.rb` prior to running it.

I can remove it if you'd like.

Reply:
> unless you want to keep `.gitignore` as barebones as possible

Yes, that's the aim.

> A safety-conscientious individual may want to inspect `uncrustify.rb` prior to running it.

They can easily delete it afterwards :smile: